libvisual/libvisual/lv_actor.h

```diff
@@ -52,6 +52,15 @@ namespace LV
   {
   public:
 
+      /**
+       * Determines if an actor by the given name is available.
+       *
+       * @param name Name of plugin to load
+       *
+       * @return True if an actor by that name is available, else false
+       */
+      static bool available (std::string const& name);
+
       /**
        * Creates a new Actor with a plugin of a given name.
        *
```

Comment:
Having this is a good idea. Can we move this into a separate patch and add the method to `LV::Morph` and `LV::Input` as well?

Reply:
@kaixiong done :+1: